### DISCUSSION TITLE
ask for playble game list

### DISCUSSION THREAD

**Discussion by masterpiece67** [skyline-emu/skyline#1323] [Game Requests]
![Screenshot_2022-05-16-00-04-30-223_com lonelycatgames Xplore](https://user-images.githubusercontent.com/92961132/168482397-e1bf3141-2e04-43c8-9018-31ab6ac51ab4.jpg)

Dev, How many games can be played in the skyline app? If so, can we ask for the latest compability 
game list document?

**Comment by hope3x** [comment]
https://github.com/skyline-emu/skyline-games-list/issues

There you go. You can close this as finished.

**Reply by masterpiece67** [reply]
Thanks
